holdings_maintenance/pds4/pds4checksums.py

```
@@ -0,0 +1,895 @@
+            if abspath.endswith('/.DS_Store'):      # skip .DS_Store files
+                logger.ds_store('.DS_Store skipped', abspath)
+                continue
+
+            if '/._' in abspath:                    # skip dot-underscore files
+                logger.dot_underscore('._* file skipped', abspath)
+                continue
+
+            if '/.' in abspath:                     # flag invisible files
+                logger.invisible('Invisible file', abspath)
+
+            f.write('%s  %s\n' % (hex, abspath[lskip:]))
```

Ruff (0.14.5)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)